docs/examples/add-to-your-application/python/app/data_filtering.py

```diff
@@ -0,0 +1,82 @@
+from .routes import serialize, app
+from . import models
+
+from oso import Oso
+
+from sqlalchemy import Column, String, Boolean
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+
+# please forgive this nightmare -dave
```

Comment:
🤔

Reply:
this file is a mess because I have an `oso` instance in here, and one in `oso.py`. This also adds a route to the flask app. I structured this more around making the docs snippets work than the code making sense since we aren't making this downloadable (the quickstart is downloadable).

I guess a comment to this effect would be better...